src/q3bsp/Q3BspLoader.hpp

```diff
@@ -2,6 +2,7 @@
 #define Q3BSPLOADER_INCLUDED
 
 #include "q3bsp/Q3BspMap.hpp"
+#include <fstream>
```

Comment:
Is this include needed here?

Reply:
GCC doesn't like fstream included in the source file for some reason.
I removed from the source, now its included only in the header.